### ISSUE TITLE
Colors in detailed test report are confusing or not correct

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-gml32#17]
Left side all the block icons are in red, they should be green or yellow.

TE 4.4
GML 3.2.1 r 1 .22
RI Tested:
http://services.interactive-instruments.de/cite2015/simpledemo/cgi-bin/cities-postgresql/wfs?request=GetFeature&service=WFS&version=2.0.0&TYPENAMES=ci:City&COUNT=10&NAMESPACES=xmlns(ci,http://www.interactive-instruments.de/namespaces/demo/cities/2.0/cities)

![gml-result](https://cloud.githubusercontent.com/assets/3738898/11578373/df9a3262-99f2-11e5-9528-70b703af4d03.jpg)

**Comment by rjmartell** [comment]
Here's what we might expect in the results frame on the left:

* no test failures: GREEN
* one or more tests failed: RED
* all tests skipped: YELLOW

The HTML report is generated using the testng-report.xsl stylesheet in the `teamengine-spi` module, so this is not really an issue about the GML test suite. The colored background arises from an empty div element with a class attribute: testCaseFail, testCasePass, testCaseSkip.

```
.testCaseFail, .testCasePass, .testCaseSkip { padding-right: 15px; width: 15px; height: 15px; margin: 2px 4px 2px 2px; display: inline; }
.testCaseFail { background-color: red; }
.testCasePass { background-color: green; }
.testCaseSkip { background-color: yellow; }
```

**Comment by rjmartell** [comment]
The given request URI doesn't resolve: An error code (400) with an exception report is returned:

    Request rejected due to errors.
    InternalExceptionCode: iiOperationParsingFailed.  
    Reason: The requested FeatureType "cix:City" is not available for the WFS.

The namespace is incorrect. According to the capabilities document it should be "http://www.interactive-instruments.de/namespaces/demo/cities/4.0/cities".

**Comment by bermud** [comment]
@keshav-nangare, please take a look at this issue and make sure that the new HTML is getting the colors correct.

**Comment by dstenger** [comment]
All config tests should be ignored when the color of the report is determined.
This is fixed in https://github.com/opengeospatial/teamengine/issues/266

After the fix has been done, above described tests have to be updated (https://github.com/opengeospatial/ets-gml32/issues/17#issuecomment-165176029):
* GML application schemas defining time:  Config test must be made to a real test.
* GML application schemas defining spatial topologies: Config test must be made to a real test.
* GML Documents: Should already work as expected.

**Comment by dstenger** [comment]
Behaviour can be reproduced with: http://cite.deegree.org/deegree-webservices-3.4.0/services/gml321?service=WFS&request=GetCapabilities

See following screenshot:
![ets-gml321](https://user-images.githubusercontent.com/2989166/43461533-09831e7a-94d4-11e8-8d93-663c31193500.png)

All spatial topology tests must be skipped as there are not data with GML topology.

**Comment by keshav-nangare** [comment]
@dstenger 

As we discussed the `configurationPolicy `attribute from the testng.xml, if we change the value to "`skip`" then it will skipping all the test but not changing the status of the configuration test. So this solution is not working.

Proposal: We had to add manually skipException into all configuration test then we get expected result(i.e. skip) as similar to the `GML Document` tests.

I have tested with the `GML application schemas defining time` with the test `HasTopologyComponents`.

See following screenshot with the expected behaviour:

![gml32_skip_config_test](https://user-images.githubusercontent.com/14732650/43532190-25cb2d24-95cf-11e8-9353-bb57441585e3.png)

**Comment by keshav-nangare** [comment]
> - GML application schemas defining time: Config test must be made to a real test, all other test must depend on this test and the result of this test must be a skip.
> - GML application schemas defining spatial topologies: Config test must be made to a real test, all other test must depend on this test and the result of this test must be a skip.
> - GML Documents: Should already work as expected.


Updated the above configuration test as per the proposal.

Fixed with commit: b0be780 [PR: #30]
